src/client/app/components/search/searchResults.js

```diff
@@ -28,15 +29,22 @@ const SearchResults = ({ searchResults, playlistId, dispatch }) => {
                 className="search-results__grid-wrapper"
             >
                 {searchResults && searchResults.map((tile, i) => {
-                    const { artist, title, album, images } = tile;
+                    const { artist, title, album, images, releaseDate } = tile;
                     const image = images.length && images[1] ?
                         images[1] : { url: 'http://placehold.it/300x300' };
 
+                    let detail = <ResultDetail
```

Comment:
A few things: `const` is preferable whenever a variable is not reassigned and in this case we actually don't need to create a variable since we can inline the component. Also, components without children are self-closing, e.g.
```
<ReleaseDetail
    artist={artist}
    album={album}
    releaseDate={releaseDate}
/> 
```
It's also a convention that all React components have capital names (so in this case `Detail` would satisfy the convention.